FirebaseUI/Database/API/FirebaseArray.h

```diff
@@ -51,6 +51,16 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property(strong, nonatomic) NSMutableArray<FIRDataSnapshot *> * snapshots;
 
+/**
+ * The number of objects in the FirebaseArray.
+ */
+@property(nonatomic, readonly) NSUInteger count;
+
+/**
+ * The items currently in the FirebaseArray.
+ */
+@property(nonatomic, readonly, copy) NSArray *items;
```

Comment:
Wanted to expose a plain immutable data type since `FirebaseArray` is full of behavior despite pretending to be an array.